Description: Drop broken Python 2 detection
 The Python 2 detection is broken for the case that Python isn't even
 installed (e.g. because no Python support needs to be built) -- so broken that
 it causes the build to fail.  As we don't intend to ship Python 2.x libraries
 (see Debian Abug #936886), let's just rip out the entire detection logic.
Author: Micha Lenk <micha@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/936886
Forwarded: no
Last-Update: 2019-12-04

--- libktoblzcheck-1.50.orig/CMakeLists.txt
+++ libktoblzcheck-1.50/CMakeLists.txt
@@ -70,23 +70,6 @@ if(CMAKE_USE_PTHREADS_INIT)
     set(HAVE_LIBPTHREAD 1)
 endif()
 
-find_package(Python2 COMPONENTS Interpreter Development)
-# for cmake < 3.12
-if(NOT Python2_Interpreter_FOUND)
-    find_package(PythonInterp 2.7 REQUIRED)
-    set(Python2_Interpreter_FOUND TRUE)
-    set(Python2_EXECUTABLE ${PYTHON_EXECUTABLE})
-endif()
-if(Python2_Development_FOUND)
-    string(REGEX REPLACE "^([0-9]+.[0-9]+).*" "\\1" Python2_VERSION_MAJOR_MINOR ${Python2_VERSION})
-else()
-    find_package(PythonLibs 2.6 REQUIRED)
-    set(Python2_Development_FOUND TRUE)
-    set(Python2_VERSION ${PYTHONLIBS_VERSION_STRING})
-    string(REGEX REPLACE "^([0-9]+.[0-9]+).*" "\\1" Python2_VERSION_MAJOR_MINOR ${Python2_VERSION})
-    set(Python2_SITEARCH "${CMAKE_INSTALL_LIBDIR}/python${Python2_VERSION_MAJOR_MINOR}/site-packages")
-endif()
-
 ########### configure checks ###############
 include(CheckIncludeFile)
 check_include_file(dirent.h     HAVE_DIRENT_H)
